financial-connections/src/test/java/com/stripe/android/financialconnections/repository/TestConsumerSessionRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.stripe.android.financialconnections.repository
+
+import com.stripe.android.model.ConsumerSession
+
+internal class TestConsumerSessionRepository : ConsumerSessionRepository {
```

Comment:
Do we need this? The `RealConsumerSessionRepository` only need a `SavedStateHandle` (which you can easily construct), so I feel like we should use that instead.

Then we also don’t need the public `toCached`.

Reply:
you're totally right! https://github.com/stripe/stripe-android/pull/8967/commits/41995349b6d9a9ee7f3bff820a821a00f61e2a15